script.js

```diff
@@ -1,21 +1,48 @@
 const Books = JSON.parse(localStorage.getItem('booksData')) || [];
 
-const addBook = () => {
-  const title = document.getElementById('title').value;
-  const author = document.getElementById('author').value;
-  const book = { id: new Date().getTime(), title, author };
-  if (Books !== null) {
-    Books.push(book);
-    const convertedBooks = JSON.stringify(Books);
-    localStorage.setItem('booksData', convertedBooks);
-    window.location.reload();
-  } else {
-    Books.push(book);
-    const convertedBooks = JSON.stringify(Books);
-    localStorage.setItem('booksData', convertedBooks);
-    window.location.reload();
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.isbn = id;
   }
-};
+
+  addBook = () => {
+    const title = document.getElementById('title').value;
+    const author = document.getElementById('author').value;
+    const book = { id: new Date().getTime(), title, author };
```

Comment:
- [ ] The id here is to push a number to each book that has an ID like 0 => to number.